frontend/src/pages/Player/RightPlayerPanel/components/NetworkResourcePanel/NetworkResourceLogs.tsx

```diff
@@ -40,14 +40,20 @@ export const NetworkResourceLogs: React.FC<{
 	}>()
 	const requestId = resource.requestResponsePairs?.request?.id
 	const [query, setQuery] = useState('')
-	const startDate = useMemo(
-		() => new Date(sessionStartTime + resource.startTime - TIME_BUFFER),
-		[sessionStartTime, resource.startTime],
-	)
-	const endDate = useMemo(
-		() => new Date(sessionStartTime + resource.responseEnd + TIME_BUFFER),
-		[resource.responseEnd, sessionStartTime],
-	)
+	const startDate = useMemo(() => {
+		// startTime used in highlight.run <8.8.0 for websocket events and <7.5.4 for requests
+		const resourceStart =
+			resource.startTimeAbs ?? sessionStartTime + resource.startTime
```

Comment:
just checking the order of operations here
this is the same as `resource.startTimeAbs ?? (sessionStartTime + resource.startTime)` right?

Reply:
Yep! Same think and the linter actually removes the `()` on save